app/forms/idv/state_id_form.rb

```diff
@@ -3,9 +3,9 @@ class StateIdForm
     include ActiveModel::Model
     include FormStateIdValidator
 
-    ATTRIBUTES = %i[first_name last_name dob state_id_address1 state_id_address2
-                    state_id_city state_id_zipcode state_id_jurisdiction
-                    state_id_state state_id_number same_address_as_id].freeze
+    ATTRIBUTES = %i[first_name last_name dob identity_doc_address1 identity_doc_address2
+                    identity_doc_city identity_doc_zipcode state_id_jurisdiction
+                    identity_doc_state state_id_number same_address_as_id].freeze
```

Comment:
looking at the ticket again i see we didn't include anything about `state_id_number` but I'm wondering if it should be updated as the id used may not be a state id. Would be a good thing to check w/ prod about.

Reply:
I think this might be a similar situation to [the reply I sent to Matt](https://github.com/18F/identity-idp/pull/8235/files#r1170594134), where we plan to wait to change this until the feature to use multiple types of ID is implemented.

Do you mind clarifying what you mean by check with prod?

Reply:
oh i meant to ask one of the product folks if this is a field that should potentially also be updated.

Reply:
Thank you for clarifying, @svalexander ! I reached out to David and he confirmed that `state_id_number` should be changed to `identity_doc_number`. I will work on that shortly.

Reply:
Sorry for the double post - after talking a little further with David, we're actually still a little unsure about this one, and I am going to ask in Slack.

Reply:
See [discussion in Slack](https://nava.slack.com/archives/C038SAM1F18/p1681921642119969); @eileen-nava says that this change is out of scope for this ticket